tests/e2e/features/actions/mainPage/faucet/actions-faucet.feature

```diff
@@ -0,0 +1,35 @@
+@action @smoke @regression @faucet @emptyWallet
+Feature: Faucet
+
+  Background:
+    Given Connect Metamask extension with login action
+
+  @id1550 @mainnet
+  Scenario: Check Faucet NOT available on Mainnet
+    Given A wallet should be "empty"
+    When I go to page "/?network=era-mainnet"
+    Then Element with "text" "Not enough tokens?" should be "invisible"
+    Then Element with "text" "Use official zkSync Era faucet" should be "invisible"
+    Then Element with "class" "default-button variant-primary destination-item-button" should be "invisible"
+    Then Element with "class" "content-block-container faucet-notification" should be "invisible"
+
+  @id1635
+  Scenario: Check Out of funds Faucet page
+    Given A wallet should be "empty"
+    When I go to page "/transaction/zksync/era/faucet/?network=era-goerli"
+    Then Element with "partial class" "lottie-animation-container" should be "visible"
+    Then Element with "text" "Out of funds" should be "visible"
+    Then Element with "text" " The faucet is " should be "visible"
+    Then Element with "text" "out of funds currently" should be "visible"
+    Then Element with "text" ". If you want to test your app, consider testing it locally with access to rich wallets " should be "visible"
+    Then Element with "text" "following the instructions from our docs" should be "visible"
```

Comment:
I suppose it is good enough here to match this field by some key words (not by each splitted set of words)

Reply:
Fixed in latest PR